genmc-sys/build.rs

```diff
@@ -0,0 +1,200 @@
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+
+const GENMC_LOCAL_PATH_STR: &str = "./genmc/";
```

Comment:
```suggestion
const GENMC_LOCAL_PATH: &str = "./genmc/";
```
Please also add a doc comment.